presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix IS DISTINCT FROM for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve read-path to better estimate the size of row. Output (sum) string stat even if min/max values are too long.
+* Improve performance of array_join.
+* Improve efficiency of ST_Buffer. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances. https://prestodb.io/docs/current/functions/geospatial.html.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Use JTS instead of Esri for many geometrical operations to improve efficiency and fix incorrect calculation of extreme points in certain cases.
+  Polygon WKTs are now required to have closed loops, whereas previously Esri would close the loops for you.
+  Certain other invalid geometries will also fail to be created from WKTs, such as `LINESTRING(0 0, 0 0, 0 0)`.
+  Returned WKTs may have a different point order than before.
+* Add support for get partition names by filter.
+* Add forms of approx_percentile accepting an accuracy parameter.
+* Add new session property "aggregation_partitioning_merging_strategy" to control merging partitioning preference when adding repartition remote exchange around aggregation node. Default option is `LEGACY` and can be overwritten to `TOP_DOWN` or `BOTTOM_UP`.
+* Add IGNORE NULLS clause to various window functions. https://prestodb.io/docs/current/functions/window.html.
+
+Hive Changes
+____________
+* Allow reading data from HDFS while caching the fetched data on local disks. Turn on the feature by specifying the cache directory config `cache.base-directory`.
+* Add support for parallel partition fetching for the Glue metastore.
+* Add support for the AWS Default Credentials Provider for s3.
```

Comment:
Is the convention uppercase S3 or lowercase s3?

Reply:
it should be S3